orderbook/src/bin/orderbook.rs

```diff
@@ -0,0 +1,64 @@
+use actix::Actor;
+use actix_web::middleware;
+use actix_web::web;
+use actix_web::App;
+use actix_web::HttpServer;
+use diesel::r2d2;
+use diesel::r2d2::ConnectionManager;
+use diesel::PgConnection;
+use orderbook::routes::delete_order;
+use orderbook::routes::get_order;
+use orderbook::routes::get_orders;
+use orderbook::routes::index;
+use orderbook::routes::post_order;
+use orderbook::routes::price_feed_route;
+use orderbook::routes::update_order;
+use orderbook::routes::websocket;
+use orderbook::run_migration;
+use std::sync::atomic::AtomicUsize;
+use std::sync::Arc;
+use tracing::level_filters::LevelFilter;
+
+#[actix_web::main]
+async fn main() -> std::io::Result<()> {
+    orderbook::logger::init_tracing(LevelFilter::DEBUG, false).unwrap();
+
+    // set up database connection pool
+    let conn_spec = "postgres://postgres:mysecretpassword@localhost:5432/orderbook".to_string();
```

Comment:
"mysecretpassword" should contain a special character, uppercase, and number - try "My$3cretpassword" :wink:

Reply:
😅